page/using-jquery-core/manipulating-elements.md

```diff
@@ -132,23 +132,33 @@ myList.append( myItems.join( "" ) );
 
 ## Manipulating Attributes
 
-jQuery's attribute manipulation capabilities are extensive. Basic changes are simple, but the `.attr()` method also allows for more complex manipulations. It can either set an explicit value, or set a value using the return value of a function. When the function syntax is used, the function receives two arguments: the zero-based index of the element whose attribute is being changed, and the current value of the attribute being changed.
+jQuery's attribute manipulation capabilities are extensive.
+Basic changes are simple, but the `.attr()` method also allows for more complex manipulations.
```

Comment:
Probably this line and the next can be combined with the previous one. I'm not sure we need break lines in between.